### PR TITLE
Pip install, basic support (sashafrey)

### PR DESCRIPTION
**Pull request by sashafrey**

### CODE REVIEW

**Comment by JeanPaulShapo** [comment]
@sashafrey You might want to edit also `python/artm/wrapper/api.py`, `setup.py` and `python/setup.py`, because right now they configured to expect shared library object in `python/artm/wrapper` folder.

**Comment by sashafrey** [comment]
@JeanPaulShapo For Windows this is covered by ``set PATH=%PATH%;C:\BigARTM\bin`` (a step from windows installation guide: http://docs.bigartm.org/en/stable/installation/windows.html). From what I know adding ``bin`` to ``PATH`` is enough to make ``cdll = ctypes.CDLL(lib_name)`` works. Windows scans all ``PATH`` when tries to load a``.dll``.

As for ``setup.py`` and ``python/setup.py`` -- I don't think they are used on Windows. At least these is not mentioned in the installation guide for Windows.

File: MANIFEST.in [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Manifest template
```
Comment (sashafrey) [inline MANIFEST.in]: @JeanPaulShapo What's the purpose of MANIFEST.in file?

Comment (JeanPaulShapo) [inline MANIFEST.in]: @sashafrey It is for specifying which files are to include in source distribution.

**Comment by JeanPaulShapo** [comment]
@sashafrey By the way, is it OK to have `artm.dll` in `python/artm/wrapper` folder, if there were platform-independent `cp` in CMakeLists.txt?

**Comment by sashafrey** [comment]
@JeanPaulShapo To me this looks somewhat strange, because it copies build artifacts outside of the ``build`` folder. I think this should only happen if we install things, but during normal build this is odd. However for now I'm too happy to see ``pip install``, so if you need that to make it work -- then fine, keep it as it is :)

In general I'm trying to know what folders I need to delete in order to make a clean build (not just for bigartm, but in general --- for any code that I build). For Ideally it should be sufficient to remove the folder where we run ``cmake`` (e.g. ``build``). Currently we also output into ``python/build`` -- but this is probably OK, and I'm sure there are other projects that have specific ``build`` folder specific to python code.

**Comment by sashafrey** [comment]
Closing this PR as https://github.com/bigartm/bigartm/pull/597/commits/c40316345665bca2c7f0049b724e37e7919823e3 is a better fix for the issue.

**Pull request closed without merging**